src/fpd/enrichment.js

```diff
@@ -78,6 +98,18 @@ function removeUndef(obj) {
   return getDefinedParams(obj, Object.keys(obj))
 }
 
+export function tryToGetCdepLabel(cb = getCookieDeprecationLabel) {
+  return GreedyPromise.resolve(isActivityAllowed(ACTIVITY_ACCESS_DEVICE, activityParams(MODULE_TYPE_PREBID, 'cdep')) && cb());
+}
+
+function getCookieDeprecationLabel() {
```

Comment:
I'll be more specific - this function, as far as I can tell, behaves the same as directly calling `navigator.CookieDeprecationLabel.getValue()` except that you lose rejections / errors, the promise you return never completes in that case.